Nef_3/include/CGAL/Nef_3/SNC_k3_tree_traits.h

```diff
@@ -100,23 +101,13 @@ class ComparePoints<CGAL::Lazy_kernel<EK>, Coordinate> {
 
 template <class SNC_decorator>
 class Side_of_plane {
-public:
-  typedef typename SNC_decorator::SNC_structure SNC_structure;
+
   typedef typename SNC_decorator::Decorator_traits Decorator_traits;
 
   typedef typename Decorator_traits::Vertex_handle Vertex_handle;
   typedef typename Decorator_traits::Halfedge_handle Halfedge_handle;
   typedef typename Decorator_traits::Halffacet_handle Halffacet_handle;
 
-#ifdef CGAL_NEF3_TRIANGULATE_FACETS
```

Comment:
Hi Giles, I guess you are not really happy that we are lagging behind with approving and merging your tsunami of pull requests.  We really appreciate them and we allocate resources where we find time. 
The problem with this PR is that it seems to combine several others in the diff. That makes it pretty hard to review.  I suggest that we deal with the PRs in order and when an older one  (e.g. the one on "triangulate faces") gets merged into mater this block should be no longer in the diff.

Reply:
Yeah, I was finding it tricky to manage. If you are happy to hold off fully reviewing this until #5740 is merged, I can rebase and resolve conflicts.